docs/user-guide/my-time-tracking/README.md

```diff
@@ -0,0 +1,110 @@
+
+![My tracking page overview in OpenProject](openproject_user_guide_my_time_tracking_module_overview.png)
+
+### My time tracking page filters
+
+1. On the *My time tracking* page, you’ll find the following interface elements:
+   1. **Zen mode** button — toggles a distraction-free view.
+   2. **Tab selector** — switch between *Calendar* and *List* views.
+   3. **Zoom level dropdown menu** — choose the time scale for the view. Following options are available:
+      - Day
+      - Work Week
+      - Week
+      - Month
+   4. **Displayed date range** — shows the currently visible timeframe.
+   5. **Navigation arrows** — move to the previous or next time period.
+   6. **Today** button — jump directly to the current day.
+   7. **+ Log time** button — add a new time entry.
+
+
+
+![My time tracking module filters explained](openproject_user_guide_my_time_tracking_module_filters_explained.png)
+
+### Calendar view
+
+The *Calendar* view opens by default if start and finish times for time entries have been enabled in [System Administration](../../system-admin-guide/time-and-costs). You can adjust the zoom level using the dropdown menu, with options for **Day**, **Work week**, **Week**, or **Month**.
+
+Existing time entries appear on the corresponding days they were logged. Entry colors reflect the type of work package associated with the logged time.
+
+You can log time by selecting a time range anywhere on the calendar or by using the **+ Log time** button.
+
+You can drag and drop time entries to reposition them within the same day or move them to a different day. To adjust the duration of an entry, drag the top or bottom edge of the time entry up or down.
+
+![Example of logging time in OpenProject My time tracking module](openproject-my-time-tracking-page.gif)
+
+To edit a time entry, click it in the calendar, make your changes, and save. You can also delete it from the same menu.
+
+At the bottom of each day a sum of hours per day is displayed. If week or month is selected, a respective total is also shown in the bottom right corner. 
+
+![Sums of hours displayed at the bottom of the Calendar tab under My time tracking module in OpenProject](openproject_user_guide_my_time_tracking_module_calendar_tab_total_sums.png)
+
+### List view
+
+The *List* view opens by default if start and finish times for time entries are **disabled** in [System Administration](../../system-admin-guide/time-and-costs). Just like in the *Calendar* view, you can select the zoom level using the dropdown menu — options include **Day**, **Work week**, **Week**, or **Month**.
+
+In this view, time entries are grouped by day. Each day displays the total number of entries and their combined duration on the right side of the header line.
+
+- When the **Day** zoom is selected, the view is expanded by default.
+
+- When **Week** or **Month** zoom is selected, each day is collapsed by default but can be expanded to view details.
```

Comment:
In the week and month views, not all the days/week are collapsed by default. Only the past ones are collapsed. The current day of the week or current week is expanded, and all future dates also.